BSPConversionCmd/Program.cs

```diff
@@ -22,38 +22,77 @@ class Options
 			[Option("prefix", Required = false, HelpText = "Prefix for the converted BSP's file name.")]
 			public string Prefix { get; set; }
 
-			[Value(0, MetaName = "input file", HelpText = "Input Quake 3 BSP/PK3 to be converted.", Required = true)]
-			public string InputFile { get; set; }
+			[Option('i', "input", Required = false, HelpText = "Input file(s) and its path to be processed. (i.e. -input \"C:\\path\\to\\file.pk3)\" ", Separator = ',')]
```

Comment:
If you use Value instead of Option, you can drag and drop the PK3 files onto BSPConv without needing the -i argument.
`[Value(0, MetaName = "input files", Required = true, HelpText = "Input Quake 3 BSP/PK3 files to be converted.")]`